database/seeds/CurrencyTableSeeder.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Illuminate\Database\Seeder;
```

Comment:
We don't need the seeder at all now actually. You can remove this file entirely.